src/components/App.tsx

```diff
@@ -9,7 +9,12 @@ const App: React.FunctionComponent = () => {
     <>
       <Header />
       <Routes>
-        <Route path="/" element={<LandingPage />} />
+      {/* under review */}
+        <Route path="/" element={<Home />} />
```

Comment:
As the Home component does not exist yet we can just use a placeholder element here eg.
`<div>Homepage</div>`